rcfile

```diff
@@ -12,7 +12,8 @@ init-hook='
     sys.path.append("./tmcprototype/DishLeafNode");
     sys.path.append("./tmcprototype/SubarrayNode");
     sys.path.append("./tmcprototype/CentralNode");
-    sys.path.append("/usr/local/lib/python3.5/dist-packages")'
+    sys.path.append("/usr/local/lib/python3.5/dist-packages");
```

Comment:
I believe there is better way of adding default path of dist-packages directory. But This is fine for me.

Reply:
Implemented as per our discussion. :)